frontend/views/containers/contributions/IncomeDetails.vue

```diff
@@ -0,0 +1,239 @@
+<template lang='pug'>
+modal-base-template(ref='modal')
+  .wrapper-container
+    .c-content
+      i18n.c-title(tag='h2') Income Details
+
+      form.card.c-card(
+        @submit.prevent='submit'
+        novalidate='true'
+      )
+        fieldset.c-firstQuestion(v-error:incomeDetailsKey='{position: "beforeend"}')
+          legend.label
+            | {{ L('Do you make at least {groupMincomeFormatted} per month?', { groupMincomeFormatted }) }}
```

Comment:
Why the sudden switch to `L` from `i18n` here?

Reply:
Because it has a component as children, and i18n doesn't support it.